### ISSUE TITLE
Please support (or remove) Linux support

### ISSUE THREAD

**Issue by j2l** [Wano-k/RPG-Paper-Maker#111]
Sorry for the clickbait title.
Last version of needed dependencies are for ubuntu 16 (2016), yes 8 years ago: https://www.ubuntuupdates.org/package/kubuntu-ppa_backports/xenial/main/base/qtdeclarative5-qtmultimedia-plugin
without it, you simply can't run `run.sh`
That's not what is called Linux support.
It's like being mandatory to use Windows XP (support ended in 2016) to run something you bought in 2024. 8y old, this is a breaking record.
I really hope you have or develop a fix quick instead of dropping Linux.
Developing on linux is not the same as targeting linux players.

**Comment by Wano-k** [comment]
Yes the project is starting to get old and tech was not updated for linux. There is the 3.0.0 that is completely rebuilding everything and will care a lot about keeping everything up-to-date. Until then the support of linux is paused for some months!
